source/Halibut.sln

```diff
@@ -70,18 +72,6 @@ Global
 		{65BD1243-FCB9-40C6-A926-8B47E10904AA}.Release|Mixed Platforms.Build.0 = Release|Any CPU
 		{65BD1243-FCB9-40C6-A926-8B47E10904AA}.Release|x86.ActiveCfg = Release|Any CPU
 		{65BD1243-FCB9-40C6-A926-8B47E10904AA}.Release|x86.Build.0 = Release|Any CPU
-		{66014922-3A83-42A2-8C23-D327160175C9}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
The new project change above is needed, but these look like the accidental changes we VS uses have to look out for?

Reply:
That's normally `55223D02-0CE9-4325-9A4D-5E88A544B633`

`66014922-3A83-42A2-8C23-D327160175C9` just seemed to be an orphaned configuration e.g. there is no related project

Reply:
Cool